tools/webhook-router/Azure.Sdk.Tools.WebhookRouter/Routing/Router.cs

```diff
@@ -160,7 +160,7 @@ private async Task<byte[]> ReadAndValidateContentFromGitHubAsync(GitHubRule rule
             return payloadContent;
         }
 
-        private SHA256CryptoServiceProvider sha256 = new SHA256CryptoServiceProvider();
+        private SHA256 sha256 = SHA256.Create();
```

Comment:
In net8 the SHA256CryptoServiceProvider class is deprecated and they recommend using the SHA256CryptoServiceProvider's base class' Create method.